pkg/agent/cniserver/pod_configuration.go

```diff
@@ -104,19 +104,30 @@ func newPodConfigurator(
 	}, nil
 }
 
-func findContainerIPConfig(ips []*current.IPConfig) (*current.IPConfig, error) {
+func findContainerIPConfigs(ips []*current.IPConfig) ([]*current.IPConfig, error) {
+	var ipcs []*current.IPConfig
 	for _, ipc := range ips {
 		if ipc.Version == "4" {
```

Comment:
Can there be another possible value beside 4 and 6? If not do we need to check Version at all?

Reply:
Right, it's either 4 or 6. I will remove this check, thanks.